batch_job/src/work_queue_factory.c

```diff
@@ -360,7 +362,7 @@ static int submit_worker( struct batch_queue *queue )
 		cmd = newcmd;
 	}
 
-	char *files = string_format("work_queue_worker");
+	char *files = os ? string_format("") : string_format("work_queue_worker");
```

Comment:
I would make it:

files = os ? string_format("runos.py") : string_format("work_queue_worker");

Ideally, the factory should not send anything that depends on a global path.

Reply:
That is, it should work as if we passed --wrapper-file runos.py to the factory.